language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/BallerinaTextDocumentService.java

```diff
@@ -528,7 +528,7 @@ void setClientCapabilities(LSClientCapabilities clientCapabilities) {
                 String formattedSource = Formatter.format(syntaxTree).toSourceCode();
 
                 LinePosition eofPos = syntaxTree.rootNode().lineRange().endLine();
-                Range range = new Range(new Position(0, 0), new Position(eofPos.line(), eofPos.offset()));
+                Range range = new Range(new Position(0, 0), new Position(eofPos.line() + 1, eofPos.offset()));
```

Comment:
is this the correct place to `+1` the range? Are we getting expected result when formatted through CLI?

Reply:
Yes. 
`Formatter.format(syntaxTree)` returns the same result.